content/en/docs/concepts/containers/images.md

```diff
@@ -157,6 +157,30 @@ that Kubernetes will keep trying to pull the image, with an increasing back-off
 Kubernetes raises the delay between each attempt until it reaches a compiled-in limit,
 which is 300 seconds (5 minutes).
 
+## Serial and parallel image pulls
+
+By default, kubelet pulls images serially. In other words, kubelet will send only
+one image pull request to the image service at a time. Other image pull requests
+have to wait until the one being processed is complete.
+
+If you would like to enable parallel image pulls, you can set the field
+`serializeImagePulls` to false in kubelet configuration. With `serializeImagePulls`
+set to false, image pull requests will be sent to the image service immediately,
+and multiple images will be pulled at the same time.
+
+### Maximum parallel image pulls
+{{< feature-state for_k8s_version="v1.27" state="Alpha" >}}
+By default, when `serializeImagePulls` is set to false, kubelet does not limit the
+maximum number of images being pulled at the same time. If you would like to limit
+the number of parallel image pulls, you can set the field `maxParallelImagePulls`
+in kubelet configuration. With `maxParallelImagePulls` set to `N`, only `N` images
+can be pulled at the same time, and any image pull beyond `N` will have to wait
+until at least one ongoing image pull is complete.
```

Comment:
I'd suggest to mention that the same image pull request will stay in line waiting for it's turn. There is no deduplication on kubelet side

Reply:
I amended the suggestion in https://github.com/kubernetes/website/pull/39732#discussion_r1139172957 to account for this.